README.md

```diff
@@ -126,6 +126,13 @@ Default: `true`
 
 Indicates whether [Google AMP pages](https://www.ampproject.org/) should be ignored and not be added to the sitemap.
 
+### ignoreCanonacalized
```

Comment:
```suggestion
### ignoreCanonicalized
```
correct spelling throughout?

Reply:
Ha, must have added the typo'd version to the clipboard and used it throughout. Should be fixed now - thank you for catching!